CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ docker run \
   -v $(pwd):/workspace \
   -w /workspace internalpcfplatformautomation/ci:testing \
   /bin/bash \
-  -c "go install github.com/onsi/ginkgo/ginkgo && CGO_ENABLED=1 ginkgo -r -race -succinct -randomizeAllSpecs -randomizeSuites -keepGoing -skipPackage acceptance,vmlifecyclecommands,vmmanagers"
+  -c "go install github.com/onsi/ginkgo/v2 && CGO_ENABLED=1 ginkgo -r -race -succinct -randomizeAllSpecs -randomizeSuites -keepGoing -skipPackage acceptance,vmlifecyclecommands,vmmanagers"
```

Comment:
According to the [docs](https://onsi.github.io/ginkgo/), this should be `go install github.com/onsi/ginkgo/v2/ginkgo`.

Reply:
thanks for catching this